docs/Installing.asciidoc

```diff
@@ -35,23 +35,22 @@ script. It essentially automates the steps mentioned in the
 
 === Directly on your machine
 
-This should work on openSUSE Leap and openSUSE Tumbleweed and will setup openQA
-on your machine.
+On openSUSE Leap and openSUSE Tumbleweed to setup openQA on your machine
+simply download and execute the openqa-bootstrap script as root - it will do
+the rest for you:
 
 [source,sh]
 -------------------------------------------------------------------------------
-zypper in openQA-bootstrap
-/usr/share/openqa/script/openqa-bootstrap
+curl -s https://raw.githubusercontent.com/os-autoinst/openQA/master/script/openqa-bootstrap | bash -x
 -------------------------------------------------------------------------------
 
-If you happen to be using an old Leap 15.0 system which does not already have the
-openQA-bootstrap RPM in the repo you can simply download the openqa-bootstrap
-script - it will do the rest for you:
+To avoid downloading and executing the script as an alternative one can
```

Comment:
Looks like one still has to execute the script.

Reply:
I'd just say that the script is also available as an openSUSE package.

Reply:
ok, updated. The complete sentence says "The script is also available from an openSUSE package to install from:"